src/Network/WebSockets/TSSocketManager.m

```diff
@@ -26,28 +22,31 @@
 NSString *const SocketClosedNotification     = @"SocketClosedNotification";
 NSString *const SocketConnectingNotification = @"SocketConnectingNotification";
 
+// TSSocketManager's properties should only be accessed from the main thread.
 @interface TSSocketManager ()
 
-@property (nonatomic, readonly, strong) OWSSignalService *signalService;
```

Comment:
`strong` / `retain` is the default behavior for object properties, so this is redundant. 99% of the time I use `@property (nonatomic)`. The exceptions are when I make a property `weak` or `readonly`.